tests/Tunnelite.Tests/TunnelTests.cs

```diff
@@ -0,0 +1,31 @@
+using Xunit;
+
+namespace Tunnelite.Tests
+{
+    public class TunnelTests
+    {
+        [Fact]
+        public void TestTunnelConnection()
+        {
+            // Arrange
+            var tunnel = new Tunnel();
+
+            // Act & Assert
+            Assert.NotNull(tunnel);
+        }
+
+        [Fact]
+        public async Task TestTunnelStartStop()
+        {
+            // Arrange
+            var tunnel = new Tunnel();
+
+            // Act & Assert
+            await tunnel.StartAsync();
```

Comment:
Consider adding a timeout parameter to handle cases where the tunnel start operation takes too long. This can help catch potential connection issues early.